core/src/acc_text/nlg/generator.clj

```diff
@@ -40,22 +41,23 @@
                                  (format "%s = %s" (str/join ", " cs) type))))
             "lin" (map (fn [index c]
                          (let [funs (get fun c)
-                               l (str/join " | " (get lin c))]
+                               body (remove str/blank? (get lin c))
+                               l (str/join " | " (cond-> body (empty? body) (conj "\"\"")))]
                            (if (seq funs)
```

Comment:
when function has an empty body (for example, when nothing is attached to segment), will put an empty string there to not result in gf syntax error